Formula/gh.rb

```diff
@@ -3,6 +3,7 @@ class Gh < Formula
   homepage "https://github.com/cli/cli"
   url "https://github.com/cli/cli/archive/v1.5.0.tar.gz"
   sha256 "49c42a3b951b67e29bc66e054fedb90ac2519f7e1bfc5c367e82cb173e4bb056"
+  head "https://github.com/cli/cli.git", branch: "trunk"
   license "MIT"
```

Comment:
```suggestion
  license "MIT"
  revision 1
  head "https://github.com/cli/cli.git", branch: "trunk"
```
The `revision` line will need to be removed at the next version.